docs/cn/instructions/eventmesh-runtime-quickstart.zh-CN.md

```diff
@@ -57,14 +57,20 @@ sh start.sh
 ![project-structure](../../images/project-structure.png)
 
 - eventmesh-common : eventmesh公共类与方法模块
-- eventmesh-connector-api : eventmesh插件接口定义模块
-- eventmesh-connector-rocketmq : eventmesh rocketmq插件模块
+- eventmesh-connector-api : eventmesh connector插件接口定义模块
+- eventmesh-connector-plugin : eventmesh connector插件模块
```

Comment:
eventmesh-connector-api was moved into eventmesh-connector-plugin，the description and picture are not updated

Reply:
Ok, I will submit a pr to fix.

Reply:
> Ok, I will submit a pr to fix.

See #323 , i want to let the student to fix it :)

Reply:
Great